packages/server-wallet/src/onchain-service/transaction-submission.ts

```diff
@@ -0,0 +1,124 @@
+import {providers, Wallet} from 'ethers';
+import PriorityQueue from 'p-queue';
+
+import {
+  TransactionSubmissionServiceInterface,
+  MinimalTransaction,
+  TransactionSubmissionOptions,
+} from './types';
+
+// FIXME: replace with
+// import {WalletError as ChannelWalletError} from '@statechannels/server-wallet';
+import {WalletError as ChannelWalletError} from '..';
```

Comment:
I think this will actually work now based on my knowledge of yarn workspaces; no need to postpone

Reply:
It keeps failing the build for me, so I'll just leave the comments for now